frontend/src/components/pages/editOnePrize.tsx

```diff
@@ -18,18 +19,47 @@ const StyledHours = styled.p`
   margin-bottom: 40px;
 `;
 
-export default function EditOnePrize() {
+type EditPrizeProps = {
+  setPrizes: (val: (prev: Prize[]) => Prize[]) => void;
+  PORT: string;
+};
+
+export default function EditOnePrize({ setPrizes, PORT }: EditPrizeProps) {
   const [itemName, setItemName] = useState('');
   const [sponsorName, setSponsorName] = useState('');
   const [sponsorImg, setSponsorImg] = useState('');
   const { prizeId } = useParams();
+  const navigate = useNavigate();
 
   const submitEdits = () => {
     console.log(itemName);
     console.log(sponsorName);
     console.log(sponsorImg);
   };
 
+  const addToPrize = async (iName: string, sName: string, sUrl: string) => {
+    await fetch(`http://localhost:3001/prizes/${prizeId}`, {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        itemName: iName,
+        sponsorName: sName,
+        sponsorImage: sUrl,
+      }),
+    });
+  };
+
+  const loadPrizes = async () => {
+    await fetch(`${PORT}/prizes`)
+      .then((res) => res.json())
+      .then((data) => {
+        setPrizes(data);
+      })
+      .catch((err) => console.log(err));
+  };
+
```

Comment:
the patch fetch to the backend returns the updated prize, so you don't have to do an additional call to the database to get the new prizes.